common/motion-dashboard.h

```diff
@@ -0,0 +1,67 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2024 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+#include "output-model.h"
+#include "device-model.h"
+
+namespace rs2
+{
+    class motion_dashboard : public stream_dashboard
+    {
+    public:
+
+        motion_dashboard( std::string name );
+
+        // Draw Accelerate dashboard and graph's lines
+        void draw( ux_window & window, rect rectangle ) override;
+        int get_height() const override;
+        void clear( bool full ) override;
+
+        // Show radio buttons for X, Y, Z and N lines
+        void show_radiobuttons();
+        // Show slider that change pause between acceleration values that we saving.
+        void show_data_rate_slider();
+
+    protected:
+        float x_value;
+        float y_value;
+        float z_value;
+        float n_value;  // Norm ||V|| = SQRT(X^2 + Y^2 + Z^2)
+
+        float frame_rate;
+        double last_time;
+        std::map< float, double > frame_to_time;
+
```

Comment:
You are using "X", "Y", "Z", "N" constants several times in this class code, better to use actual consts:
`constexpr const std::string x_axes_name = "X";` etc..

Reply:
`constexpr` is not possible inside class. Remove `constexpr` or put the line outside class?